tt_metal/host_api.hpp

```diff
@@ -352,37 +351,47 @@ void EnqueueProgram(CommandQueue& cq, std::variant<std::reference_wrapper<Progra
 void Finish(CommandQueue& cq);
 
 /**
- * Creates a trace object which can be used to record commands that have been run. This
- * trace can later be replayed without the further need to create more commands.
- * Return value: trace
+ * Begins capture on a trace, when the trace is in capture mode all programs push into the trace queue will not be executed.
```

Comment:
push->pushed?  or maybe:
Begins capture on a trace, in capture mode programs will not be executed ?

Reply:
Resolved with 30d3f5eec6316f2b316d7442fe2409a2cd9d5635, with a better description:

"Begins capture on a trace, when the trace is in capture mode all programs pushed into the trace queue will have their execution delayed until the trace is instantiated and enqueued."